anagram.js

```diff
@@ -0,0 +1,25 @@
+/*
+Task: check is the word is anagram to another word
+ */
+const isAnagram = (word, otherWord) => {
```

Comment:
This task already exist in repo. Better to add alternative solutions in existed files